integrations/kml-parser/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
Why do you want to ignore a help file? Sensitive info?

Reply:
good question xD - That file was generated by the spring initalizer project -> removed the entry